src/SDKs/DataBox/Management.DataBox/Microsoft.Azure.Management.DataBox.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.DataBox</PackageId>
 		<Description>Microsoft Azure Management DataBox library</Description>
 		<AssemblyName>Microsoft.Azure.Management.DataBox</AssemblyName>
-		<Version>0.9.0-preview</Version>
+		<Version>1.0.0</Version>
 		<PackageTags>AzureDataBox;DataBox;</PackageTags>
 	</PropertyGroup>
```

Comment:
Please add `PackageReleaseNotes` here
An example [here](https://github.com/Azure/azure-sdk-for-net/blob/53b16a9d7954076f9002013846c4c2840cebf92b/src/AzureStack/Admin/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj#L15)

Reply:
thanks for the refered example have taken the fix